addons/stock_picking_batch/models/stock_picking.py

```diff
@@ -73,6 +73,8 @@ class StockPicking(models.Model):
         states={'done': [('readonly', True)], 'cancel': [('readonly', True)]},
         help='Batch associated to this transfer', index=True, copy=False)
 
+    batch_user_id = fields.Many2one(related='batch_id.user_id', string="Batch Responsible")
```

Comment:
After a second thought, I'm not sure to understand why you need this related field. Does the picking's `user_id` not always set every time the batch's `user_id` is modified ?
It's look like a little redundant

Reply:
Yes, but its not the other way around, you can change back the responsible of the picking and that will not affect the batch